bundle/config/mutator/resourcemutator/capture_schema_dependency_test.go

```diff
@@ -87,7 +87,7 @@ func TestCaptureSchemaDependencyForVolume(t *testing.T) {
 	assert.Equal(t, "schemaX", b.Config.Resources.Volumes["volume5"].CreateVolumeRequestContent.SchemaName)
 
 	assert.Nil(t, b.Config.Resources.Volumes["nilVolume"])
-	assert.Nil(t, b.Config.Resources.Volumes["emptyVolume"].CreateVolumeRequestContent)
+	// assert.Nil(t, b.Config.Resources.Volumes["emptyVolume"].CreateVolumeRequestContent)
```

Comment:
Dup?

Reply:
Not a dup (second line used to check inner pointer), should have been cleaned up.